src/Containers/OhmmsSoA/VectorSoaContainer.h

```diff
@@ -34,7 +34,8 @@ template<typename T, unsigned D, typename Alloc = aligned_allocator<T>>
 struct VectorSoaContainer
 {
   using AoSElement_t = TinyVector<T, D>;
-  using Element_t    = T;
+  using size_type    = size_t;
```

Comment:
why not `Value` and `Size` in line with the type naming rules?
size_type and value_type still read as variables

Reply:
Containers follow std::vector pattern to maximize compatibility

Reply:
right sorry